docs/source/graphviz.rst

```diff
@@ -0,0 +1,44 @@
+Visualize task graphs
```

Comment:
Since this basically replaces the `inspect.rst` page, should probably delete that page as well.

Reply:
I think this is the only thing remaining ^^